docs/examples/promise/03_listen_to_balance_change/index.js

```diff
@@ -32,4 +33,4 @@ async function main () {
   });
 }
 
-main().catch(console.error);
+main().catch(console.error).finally(_ => process.exit());
```

Comment:
Same comment as above - here it actually said, send something and see the balance change. Doing this will; exit immediately with only one result. (Before it has changed)